portfolio/src/main/java/com/google/sps/servlets/RandomGreetingServlet.java

```diff
@@ -33,10 +34,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     // Randomly selects the index for the greeting that'll appear on the page
     int greetingChosen = rand.nextInt(4); 
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Random Greeting</h1>");
-    response.getWriter().println("<p>" + greetings[greetingChosen] + "</p>");
+    
+
+    // Convert the server stats to JSON
+    String json = convertToJson(greetings, greetingChosen);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
+  }
 
+  private String convertToJson(String[] greets, int chosenGreet) {
```

Comment:
Thanks for pulling this into a helper method!

Reply:
No problem! I'm glad you appreciated it 😄